font/fonts.css

```diff
@@ -0,0 +1,21 @@
+@font-face {
+    font-family: 'Gotham Pro';
+    src:
+      local('Gotham Pro Bold'),
+      local('GothamPro-Bold'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5484013)